cime_config/tests.py

```diff
@@ -267,7 +279,6 @@
             "NCK.f19_g16_rx1.A",
             "SMS.ne30_f19_g16_rx1.A",
             "ERS_Ld5.T62_oQU120.CMPASO-NYF",
-            "SMS_Ld1.T62_oQU240wLI.GMPAS-IAF-DISMF",
```

Comment:
I'm removing this one because the proposed tests are better (more than a smoke test and use DIB as well).  Also, this test never tested what it was intended to because the DISMF file was not available.